core/block.js

```diff
@@ -990,10 +990,10 @@ Blockly.Block.prototype.setColour = function(colour) {
  * @throws {Error} if the block style does not exist.
  */
 Blockly.Block.prototype.setStyle = function(blockStyleName) {
-  var theme = Blockly.getTheme();
+  var theme = this.workspace.getTheme();
   if (!theme) {
```

Comment:
Do we need this check anymore? Since theme should never be able to be null?

Reply:
Removing.